build-support/bin/upload_coverage.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+export PY=${PY:-python3}
+export PYTHON_SYS_EXECUTABLE="${PY}"
+
+REPO_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && cd ../.. && pwd -P)
+# shellcheck source=build-support/pants_venv
+source "${REPO_ROOT}/build-support/pants_venv"
+
+activate_pants_venv 1>&2
+PYTHONPATH="${REPO_ROOT}/src/python:${PYTHONPATH}" pip install coveralls
```

Comment:
It's not clear why this first call needs `src/python`... the `pip install` shouldn't consume that, should it?

Reply:
Probably harmless though. I'll merge on green.